Helper/Data.php

```diff
@@ -838,4 +839,32 @@ public function allowMinicartCheckout($store = \Magento\Store\Model\ScopeInterfa
     {
         return (bool)$this->scopeConfig->getValue(self::XML_CONGIG_MINICART_CHECKOUT, $store);
     }
+
+    /**
+     * Get threshold amount below which if set bread button or checkout won't be enabled
+     *
+     * @param string $store
+     * @return int
+     */
+    public function getThresholdAmount($store = \Magento\Store\Model\ScopeInterface::SCOPE_STORE)
+    {
+        return (int)$this->scopeConfig->getValue(self::XML_CONFIG_THRESHOLD_AMOUNT);
+    }
+
+    /**
+     * Check if cost of item/cart total is equal or greater than set threshold amount
+     *
+     * @param float $cost
+     * @return bool
+     */
+    public function aboveThreshold($cost)
+    {
+        $aboveThreshold = true;
+
+        if($thresholdAmount = $this->getThresholdAmount()){
```

Comment:
is there any reason why you're assigning $thresholdAmount in an if statement condition? Is the if meant to make sure you get a value back? I think these should be broken out into two different things,, this seems sort of confusing, since first I thought you just forgot to make it ===

Reply:
Changed